saleor/order/actions.py

```diff
@@ -205,10 +209,13 @@ def clean_mark_order_as_paid(order: "Order"):
         raise PaymentError("Orders with payments can not be manually marked as paid.",)
 
 
-def fulfill_order_line(order_line, quantity):
+def fulfill_order_line(order_line, quantity, warehouse_pk):
     """Fulfill order line with given quantity."""
     if order_line.variant and order_line.variant.track_inventory:
-        decrease_stock(order_line, quantity)
+        try:
+            decrease_stock(order_line, quantity, warehouse_pk)
+        except InsufficientStock as exc:
+            raise exc
```

Comment:
Why do we have this `try..except` here? It catches an error and re-raises it without doing anything else.